components/gallery/Gallery.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div>
+        <button v-on:click="prevImg">prev</button>
+        <button v-on:click="nextImg">next</button> 
+        <img :src="images[i]" width="200" height="100">
+
+    </div>
+</template>
+
+<script>
+export default {
+  data: function () {
```

Comment:
это можно делать по разному, но  так точно не стоит - это самый длинный вариант.
можно написать
```suggestion
  data() {
```